### PR TITLE
only check pillow seq for couch pillows

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?235941
@NoahCarnahan @gcapalbo

### CODE REVIEW

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Comment by czue** [comment]
build fail likely real
```
======================================================================
ERROR: test_basic_cloudant_seq (corehq.apps.hqadmin.tests.test_utils.TestPillowCheckpointSeqStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/hqadmin/tests/test_utils.py", line 25, in test_basic_cloudant_seq
    store = PillowCheckpointSeqStore.objects.get(checkpoint_id=self.pillow.checkpoint.checkpoint_id)
  File "/vendor/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 357, in get
    self.model._meta.object_name)
DoesNotExist: PillowCheckpointSeqStore matching query does not exist.
```

File: corehq/apps/hqadmin/tests/test_utils.py [hunk 1]
```diff
@@ -16,7 +17,14 @@ class TestPillowCheckpointSeqStore(TestCase):
     dependent_apps = ['pillowtop']
 
     def setUp(self):
+        super(TestPillowCheckpointSeqStore, self).setUp()
         self.pillow = DummyPillow()
+        self.pillow_patch = patch("corehq.apps.hqadmin.utils.get_couch_pillow_instances", return_value=[DummyPillow()])
```
Comment (dimagimon) [inline corehq/apps/hqadmin/tests/test_utils.py]: E501 line too long (119 > 115 characters)

**Comment by gcapalbo** [comment]
(restarted build)

**Pull request merged**